docs/sources/mimir/references/http-api/index.md

```diff
@@ -419,6 +419,32 @@ This endpoint unregisters the ingester from the ring even if you disable `-inges
 
 This API endpoint is usually used by scale down automations.
 
+### Prepare for unregister
+
+```
+GET,PUT,DELETE /ingester/ingester/unregister-on-shutdown
+```
+
+This endpoint is used to control whether an ingester should unregister from the ring on its next termination (i.e., the next time it receives a `SIGINT` or `SIGTERM` signal).
```

Comment:
You've used i.e. correctly here but it's often confused so Grafana's style guide (an extension of Google's) encourages an alternative.

https://developers.google.com/style/word-list#ie

I took the chance to avoid some passive voice (is used).

```suggestion
This endpoint controls whether an ingester should unregister from the ring on its next termination, that is, the next time it receives a `SIGINT` or `SIGTERM` signal.
```